### ISSUE TITLE
Spring Boot 3 upgrade and support

### ISSUE THREAD

**Issue by StefanMessner** [eventuate-foundation/eventuate-cdc#146]
Hello,

is there a plan / timeline to upgade the Eventuate Framework to support Spring Boot 3? 

Currently, the strictly usage of `org.apache.kafka:kafka-clients:{strictly 3.1.2}` breakes the build since Spring upgraded to use `org.apache.kafka:kafka-clients:3.4.0`

Thanks.

**Comment by cer** [comment]
Yes. See https://github.com/eventuate-tram/eventuate-tram-core/pull/189#issuecomment-1555492513

**Comment by cer** [comment]
The focus has been on enabling Eventuate applications to support Spring Boot 3.

There's no urgent plan to upgrade the CDC to Spring Boot 3.

**Comment by Nikhil-Tailor** [comment]
Hey @cer are there current plans to upgrade eventuate-cdc to Spring Boot 3?

**Comment by sanjeevgour** [comment]
@cer As Spring Boot 2.x is out of support and, except for 2.7 that is on extended support, there are a bunch on vulnerabilities in the current version. is there a plan to upgrade to Spring Boot 3.x. If would be helpful to know the plan forward. 

Thanks in advance.
